src/App.jsx

```diff
@@ -28,6 +29,7 @@ const App = () => (
         <Footer logo={process.env.LOGO_POWERED_BY_OPEN_EDX_URL_SVG} />
       </div>
     </Router>
+    <Head />
```

Comment:
It's weird to me that `Head` element is lower than the `Footer` element. Can you help me understand the reason for this sequence?

Reply:
I agree, I fix this

Reply:
I use a component because there are specific related changes (title and favicon) and I thought was better to maintain a simple App component and because I'm using `injectIntl` (from `@edx/frontend-platform/i18n`) which wraps the component to make it possible to change the MFE name according to the language and that needs access the `IntlProvider` thai is inside `AppProvider`